### PR TITLE
Fix the filler and fill vmIOandFlowOperationsTest

### PR DESCRIPTION
**Pull request by pirapira**
This PR changes a VMTest filler so that `testeth -t VMTests --filltests` succeeds.  This PR also contains the filled VM test files.

### CODE REVIEW

File: src/VMTestsFiller/vmIOandFlowOperationsTestFiller.json [hunk 1]
```diff
@@ -1489,7 +1489,6 @@
         "expect" : {
             "0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
                 "storage" : {
-                    "0x02" : "0x23"
```
Comment (winsvega) [inline src/VMTestsFiller/vmIOandFlowOperationsTestFiller.json:1492]: why this disappeared?

Comment (pirapira) [reply src/VMTestsFiller/vmIOandFlowOperationsTestFiller.json:1492]: I don't know, maybe something changed in cpp-ethereum.

Comment (pirapira) [reply src/VMTestsFiller/vmIOandFlowOperationsTestFiller.json:1492]: I'll investigate this https://github.com/ethereum/cpp-ethereum/issues/4395

**Comment by pirapira** [comment]
I guess we should not just merge this.

**Comment by pirapira** [comment]
I investigated the issue.  https://github.com/ethereum/cpp-ethereum/issues/4395

The filler was wrong, and a recent change started to detect the mistake in the filler.  I believe this PR is the right fix.  @winsvega

**Pull request merged**